### PR TITLE
live chat state and ui added to donation

### PR DESCRIPTION
**Pull request by ajinkyaa**

### CODE REVIEW

File: packages/app/src/components/Live/DonationModal.tsx [hunk 1]
```diff
@@ -32,18 +35,19 @@ export const DonationModal = (props: Props): ReactElement => {
...
     const handleCreateDonation = () => {
         const input = {
-            amount: utils.parseEther(number),
-            content: liveChallenge?.content,
+            amount: utils.parseEther(number).toString(),
+            content: props.content,
             challenge: liveChallenge?._id
         };
+        console.log('props.content: ', props.content);
+        console.log('utils.parseEther(number): ', utils.parseEther(number));
```
Comment (ajinkyaa) [inline packages/app/src/components/Live/DonationModal.tsx:50]: @youngkidwarrior Are these consoles required? if not, I will remove it.

Comment (youngkidwarrior) [reply packages/app/src/components/Live/DonationModal.tsx:50]: Remove please thank you

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]
This is looking good @ajinkyaa 

One thing, 
please consolidate the text and donation into one component like this
Screen Shot 2021-07-27 at 5 45 38 PM

**Comment by ajinkyaa** [comment]
Screenshot 2021-07-28 at 8 38 36 PM
Screenshot 2021-07-28 at 8 38 51 PM

**Comment by ajinkyaa** [comment]
@youngkidwarrior Please check above image and let me know if you need any changes

**Review by youngkidwarrior — APPROVED** [review]

**Pull request merged**